hnn_core/network_builder.py

```diff
@@ -602,22 +602,138 @@ def aggregate_data(self):
     def state_init(self):
         """Initializes the state closer to baseline."""
 
+        # These were empirically derived from a 5 s blank simulation
```

Comment:
so this would be a "different" model than the one currently in the GUI. I propose that we keep the initialize the values similarly as before -- i.e., initialization per section not segment. With a comment how the numbers are derived and what approximation it makes and then have this dictionary thing so that it can be generalized after 0.1 release. 

I'm not against this improved approximation per se but I think it would be more valuable in the context where we can load different cell types so the more detailed initialization would help for a more detailed cell type. Otherwise, I'm not sure if it's worth deviating from the HNN-GUI model. For 0.1, I really want to be able to tell a GUI user -> hey give HNN-core a shot, it actually gives you the same output. Without them trying to hunt down tiny numerical differences.